common/src/edgeai_msc.cpp

```diff
@@ -0,0 +1,176 @@
+            multiScalerObj2.input.width = cropWidth/4;
+        else
+            multiScalerObj2.input.width = pre_proc_obj->cropWidth;
+
+        if (cropHeight/4 > pre_proc_obj->cropHeight)
+            multiScalerObj2.input.height = cropHeight/4;
+        else
+            multiScalerObj2.input.height =  pre_proc_obj->cropHeight;
+
+        multiScalerObj2.output[0].width = pre_proc_obj->cropWidth;
+        multiScalerObj2.output[0].height = pre_proc_obj->cropHeight;
+
+        resizeWidth = multiScalerObj2.input.width;
+        resizeHeight = multiScalerObj2.input.height;
+
+        tiovx_multi_scaler_module_crop_params_init(&multiScalerObj2);
+
+        multiScalerObj2.en_multi_scalar_output = 0;
+
+        useSecondaryMsc = true;
+
+    }
+
+    multiScalerObj1.num_channels = 1;
+    multiScalerObj1.num_outputs = 2;
+    multiScalerObj1.input.bufq_depth = 1;
+    for(int32_t out = 0; out < multiScalerObj1.num_outputs; out++)
+    {
+        multiScalerObj1.output[out].bufq_depth = 1;
+    }
+    multiScalerObj1.interpolation_method = VX_INTERPOLATION_BILINEAR;
+    multiScalerObj1.color_format = VX_DF_IMAGE_NV12;
+
+    multiScalerObj1.input.width = input_wd;
+    multiScalerObj1.input.height = input_ht;
+
+    multiScalerObj1.output[0].width = resizeWidth;
+    multiScalerObj1.output[0].height = resizeHeight;
+    multiScalerObj1.output[1].width = post_proc_wd;
+    multiScalerObj1.output[1].height = post_proc_ht;
+
+    tiovx_multi_scaler_module_crop_params_init(&multiScalerObj1);
+
+    multiScalerObj1.crop_params[0].crop_start_x = crop_start_x;
+    multiScalerObj1.crop_params[0].crop_start_y = crop_start_y;
+    multiScalerObj1.crop_params[0].crop_width = input_wd - (2 * crop_start_x);
+    multiScalerObj1.crop_params[0].crop_height = input_ht - (2 * crop_start_y);
+
+    multiScalerObj1.en_multi_scalar_output = 0;
+
+    LOG_DEBUG("multiScalerObj1.input.width = %d, "
+              "multiScalerObj1.input.height = %d \n",
+              multiScalerObj1.input.width, multiScalerObj1.input.height);
+    LOG_DEBUG("multiScalerObj1.output[0].width = %d, "
+              "multiScalerObj1.output[0].height = %d \n",
+              multiScalerObj1.output[0].width, multiScalerObj1.output[0].height);
+    LOG_DEBUG("multiScalerObj1.output[1].width = %d, "
+              "multiScalerObj1.output[1].height = %d \n",
+              multiScalerObj1.output[1].width, multiScalerObj1.output[1].height);
+    LOG_DEBUG("multiScalerObj1.crop_params[0].crop_start_x = %d, "
+              "multiScalerObj1.crop_params[0].crop_start_y = %d \n",
+              multiScalerObj1.crop_params[0].crop_start_x,
+              multiScalerObj1.crop_params[0].crop_start_y);
+    LOG_DEBUG("multiScalerObj1.crop_params[0].crop_width = %d, "
+              "multiScalerObj1.crop_params[0].crop_height = %d \n",
+              multiScalerObj1.crop_params[0].crop_width,
+              multiScalerObj1.crop_params[0].crop_height);
+
+    return status;
+}
+
+}
```

Comment:
Maybe add comment next to this brace that says " namespace ti::edgeai::common" so people know it is the namespace brace.

Reply:
Updated the PR. Addressed all similar issues